package.json

```diff
@@ -2,9 +2,10 @@
   "name": "nerds-oppose-society",
   "version": "0.1.0",
   "scripts": {
-    "test": "(cd backend && npm run test); (cd frontend && npm run test)",
-    "lint": "(cd backend && npm run lint); (cd frontend && npm run lint)",
-    "lint:fix": "(cd backend && npm run lint:fix); (cd frontend && npm run lint:fix)",
+    "build": "npm run build --workspaces --if-present",
+    "test": "npm run test --workspaces --if-present",
+    "lint": "npm run lint --workspaces --if-present",
+    "lint:fix": "npm run lint:fix --workspaces --if-present",
```

Comment:
this is quite cool

Reply:
Yeah it's really nice - yarn workspaces & lerna have even more powerful features along these lines but I think this works really well for our limited use case

Reply:
yeah its neat